config/routes.rb

```diff
@@ -2,6 +2,6 @@
   if Rails.env.development? && defined?(Rails::Server)
     logger = Logger.new(STDOUT)
     logger.level = Logger.const_get(::Settings.log.level_websocket.upcase)
-    mount WebsocketServer.new(:logger => logger) => '/ws'
+    mount WebsocketServer.new(:logger => logger) => '/ws/console'
```

Comment:
Please verify:

We didn't mount `ActionCable` at `/ws` before.
We only called it manually in `WebsocketServer`?

future: we are mounting websocket server at `/ws/console` and rails is auto mounting action cable at `/ws/notifications`

Reply:
:+1: